File: python/matrices.py
matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

print("\nMatriz 3x3:")

for fila in matriz:
    print(fila)

print("\nMatriz en forma de tabla:")

for fila in matriz:
    for elemento in fila:
        print(elemento, end=" ")
    print()

File: cpp/matrices.cpp
#include <iostream>

using namespace std;

int main() {

    int matriz[3][3] = {
        {1, 2, 3},
        {4, 5, 6},
        {7, 8, 9}
    };

    cout << "Matriz 3x3:" << endl;

    for (int fila = 0; fila < 3; fila++) {
        for (int columna = 0; columna < 3; columna++) {
            cout << matriz[fila][columna] << " ";
        }
        cout << endl;
    }

    cout << "Recorrido por columnas:" << endl;

for (int columna = 0; columna < 3; columna++) {
    for (int fila = 0; fila < 3; fila++) {
        cout << matriz[fila][columna] << " ";
    }
    cout << endl;
}

    return 0;
}